### PR TITLE
Cleanup some of e2e_new test code

### LINKED ISSUE #731: Cleanup the trigger and broker initialization copy-pasted code in e2e_new
Context: https://github.com/knative-sandbox/eventing-kafka-broker/pull/701#discussion_r596229236

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

Fix #731

## Proposed Changes

- :broom: Cleanup copy pasted code

### CODE REVIEW

File: vendor/knative.dev/eventing/test/rekt/resources/addressable/addressable.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	"time"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"knative.dev/pkg/apis"
+	"knative.dev/reconciler-test/pkg/k8s"
+)
+
+// Address returns a broker's address.
```
Comment (aliok) [inline vendor/knative.dev/eventing/test/rekt/resources/addressable/addressable.go:30]:
```suggestion
// Address returns an Addressable's address.
```

Comment (slinkydeveloper) [reply vendor/knative.dev/eventing/test/rekt/resources/addressable/addressable.go:30]: that's the vendor code :)

Comment (aliok) [reply vendor/knative.dev/eventing/test/rekt/resources/addressable/addressable.go:30]: LOL

Comment (aliok) [reply vendor/knative.dev/eventing/test/rekt/resources/addressable/addressable.go:30]: I haven't noticed

**Review by aliok — APPROVED** [review]
/lgtm
/approve

**Pull request merged**